docs/TargetMaintenance/CreatingAUnifiedTarget.md

```diff
@@ -4,48 +4,48 @@
 - OSD element configuration;
 - craft name;
 - rates
 
-All of these are really user preferences, and adding them as part of a Unified Target Configuration just means that most customers of this hardware will have to spend extra time on reverting the presets before they can set up their own preferences. There are no strict rules here, and what is reasonable will be dependent on what type of target it is, but manufacturers are advised to use common sense.
+All of these are really user preferences, and adding them as part of a Target Configuration just means that most customers of this hardware will have to spend extra time on reverting the presets before they can set up their own preferences. There are no strict rules here, and what is reasonable will be dependent on what type of target it is, but manufacturers are advised to use common sense.
 
-## 2. Creating a Unified Target Configuration for a new firmware
+## 2. Creating a Target Configuration for a new Board
 
-These instructions explain how to create a new Unified Target configuration without an existing Betaflight target. If your target already exists in non-unified format please skip to "Creatings a Unified Target Configuration for an existing firmware" 
-Creating Unified Targets for a new board means you must start with a generic firmware for your MCU and build up the required pin mappings. Start with a generic firmware and then validate pin mappings and hardware functionality.
+These instructions explain how to create a new Target configuration. Creating Targets for a new board means you must start with a firmware for your MCU and build up the required pin mappings. The firmware can be a generic one, or one you have built locally. Start with this firmware and then validate pin mappings and hardware functionality needed in the configuration file.
 
-### 2.1 Obtaining a unified target firmware for your MCU
+### 2.1 Obtaining a target firmware for your MCU
 
-Betaflight Configuratior distributes customised targets featuring settings specific to each board. To create firmware for a new board it is recommended to build a generic firmware for your MCU type - STM32F411, STM32F405, STM32F7X2, and so on.
-These firmware types can be created by building locally and specifying the MCU type such as `make STM32F05`. Firmware files are created in the `obj` directory and can be flashed by choosing `Load local file` in Betaflight Configurator.
+Betaflight Configurator distributes customised targets featuring settings specific to each board. To create firmware for a new board it is recommended to build a generic firmware for your MCU type - STM32F411, STM32F405, STM32F7X2, and so on. If your board has special "compile time" configuration for specific hardware or boot configuration, these flags should be added as `EXTRA_PARAMS` on the make command line (e.g. `make TARGET=STM32F405 EXTRA_PARAMS="-D'MPU_I2C_INSTANCE=I2CDEV_1'"`. The `EXTRA_PARAMS` can be considered like the old target.h file.
```

Comment:
needs '-DCLOUD_BUILD'

Reply:
> needs `-DCLOUD_BUILD`

Only if you want to simulate a cloud build. Not needed unless you use don't want to include the classic build defines.

Reply:
needs examples for:
a) #define FOO
b) #define FOO  BAR
c) #define FOO "BAR"
d) #define FOO "BAR BAR"

Reply:
Just as you would in a target.h - i.e. all the above would work.

Reply:
going by your example earlier, it appears that two-part defines need changing from 

#define FOO BAR

to

-D'FOO=BAR'

there's also the problem of defines like this, how to convert?

`#define QUADSPI1_CS_FLAGS (QUADSPI_BK1_CS_HARDWARE | QUADSPI_BK2_CS_NONE | QUADSPI_CS_MODE_LINKED)`

Reply:
it'd just like to say that on the command line it's a nightmare of character escaping...

```
-D'QUADSPI1_CS_FLAGS=\(QUADSPI_BK1_CS_HARDWARE | QUADSPI_BK2_CS_NONE | QUADSPI_CS_MODE_LINKED\)'\
```

seems to be the answer.

Reply:
and strings have to be quoted.

d)

```
-D'FOO=\"BAR\"'
```

Reply:
> ```
> -D'QUADSPI1_CS_FLAGS=\(QUADSPI_BK1_CS_HARDWARE | QUADSPI_BK2_CS_NONE | QUADSPI_CS_MODE_LINKED\)'\
> ```

but maybe not afterall...

Reply:
Not in the target configuration. Just define as per C ISO format.

The build runner converts to command line -D format.